Source/WebCore/inspector/CommandLineAPIModuleSource.js

```diff
@@ -189,6 +189,8 @@ if (inspectedGlobalObject.document && inspectedGlobalObject.Node) {
     };
 }
 
+CommandLineAPI.methods["gatherRTCLogs"] = function() { return CommandLineAPIHost.gatherRTCLogs(arguments[0]); };
```

Comment:
we may want to conditionalize this just in case `ENABLE_WEB_RTC` is false

so either this
```js
if (CommandLineAPIHost.gatherRTCLogs) {
    CommandLineAPI.methods["gatherRTCLogs"] = function(callback) {
        return CommandLineAPIHost.gatherRTCLogs(callback);
    };
}
```
or this
```js
CommandLineAPI.methods["gatherRTCLogs"] = function(callback) {
    return CommandLineAPIHost.gatherRTCLogs?.(callback);
};
```

Reply:
Done